src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,48 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp(){
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    void size() {
+        int result = numbers.size();
+        assertThat(result).isEqualTo(3);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1,2,3})
+    void contains(int number) {
+        assertThat(numbers.contains(number)).isTrue();
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = {"1:true","2:true","3:true","4:false","5:false"}, delimiter = ':')
```

Comment:
default delimiter를 사용해도 좋을 것 같습니다 🙂
```suggestion
    @CsvSource(value = {"1,true", "2,true", "3,true", "4,false", "5,false"})
```